### PR TITLE
Fix some DETECTED VIOLATIONS in the config file

### PR DESCRIPTION
**Pull request by richvdh**
consistency ftw

### CODE REVIEW

File: synapse/config/metrics.py [hunk 1]
```diff
@@ -93,10 +93,11 @@ def generate_config_section(self, report_stats=None, **kwargs):
             #known_servers: true
 
         # Whether or not to report anonymized homeserver usage statistics.
+        #
         """
 
         if report_stats is None:
-            res += "# report_stats: true|false\n"
+            res += "#report_stats: true|false\n"
```
Comment (clokep) [inline synapse/config/metrics.py:100]: The style guide seems to say that the example should provide the opposite of the default, but I see you just changed the formatting a bit here. :) So...still better.

**Comment by clokep** [comment]
The additional changes look good too! 👍

File: synapse/config/metrics.py (hunk 1 above)
Comment (richvdh) [reply synapse/config/metrics.py:100]: report_stats is odd, because it doesn't have a default.

Comment (clokep) [reply synapse/config/metrics.py:100]: Ahh, I see. 👍

**Pull request merged**